skins/oasis/css/core/ads.scss

```diff
@@ -63,7 +63,7 @@ $taboola-small-breakpoint: "#{$screen} and (max-width: 420px)";
 		width: 100%;
 	}
 
-	#HOME_TOP_LEADERBOARD, #TOP_LEADERBOARD, #CORP_TOP_LEADERBOARD, #HUB_TOP_LEADERBOARD {
+	#HOME_TOP_LEADERBOARD, #TOP_LEADERBOARD, #CORP_TOP_LEADERBOARD, #HUB_TOP_LEADERBOARD, .tmpHeader {
```

Comment:
don't call it `tmpHeader` please, is it possible to find a little bit more meaningful name?